.github/workflows/tests.yml

```diff
@@ -77,12 +77,10 @@ jobs:
                   .github/check-executable-files.sh ./
 
             - name: Check commit message
-              env:
-                  BRANCH: ${{ github.event.pull_request.base.sha }}
-                  PR_SHA: ${{ github.event.pull_request.head.sha }}
-              if: github.event.pull_request.head.sha != '' && github.event.pull_request.base.sha != ''
```

Comment:
I would go with `if: github.event.pull_request.head.sha != ''`

Reply:
Or probably just `if: env.GITHUB_EVENT_NAME == 'pull_request'`

Reply:
That might be more explicit.